### PR TITLE
WFLY-6745 SSL by default

### PR DESCRIPTION
**Pull request by stuartwdouglas**

### CODE REVIEW

**Comment by stuartwdouglas** [comment]
Retest this please

**Comment by stuartwdouglas** [comment]
Retest this please

File: feature-pack/src/main/resources/configuration/host/host.xml [hunk 1]
```diff
@@ -17,6 +17,11 @@
                 </authorization>
             </security-realm>
             <security-realm name="ApplicationRealm">
+                <server-identities>
+                    <ssl>
+                        <keystore path="application.keystore" relative-to="jboss.domain.config.dir" keystore-password="password" alias="server" key-password="password" generate-self-signed-certificate-host="localhost"/>
+                    </ssl>
+                </server-identities>
```
Comment (bstansberry) [inline feature-pack/src/main/resources/configuration/host/host.xml]: This is fine but there are also host-master.xml and host-slave.xml in this folder.

File: servlet-feature-pack/src/main/resources/configuration/standalone/template.xml [hunk 2]
```diff
@@ -18,6 +18,11 @@
                 </authorization>
             </security-realm>
             <security-realm name="ApplicationRealm">
+                <server-identities>
+                    <ssl>
+                        <keystore path="application.keystore" relative-to="jboss.server.config.dir" keystore-password="password" alias="server" key-password="password" generate-self-signed-certificate-host="localhost"/>
+                    </ssl>
+                </server-identities>
```
Comment (bstansberry) [inline servlet-feature-pack/src/main/resources/configuration/standalone/template.xml]: This is fine but there also is a host folder in servlet-feature-pack.

**Pull request merged**